### DISCUSSION TITLE
Is it possible to disable UnitOfWork implicit transaction and automatic flush?

### DISCUSSION THREAD

**Discussion by kenspirit** [mikro-orm/mikro-orm#3105] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
It looks like some unintentional change detection occurs and strange SQL is generated for flushing change.
I would like to know whether I can disable auto flushing but fully and manually control it?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by B4nan** [comment, marked as answer]
https://mikro-orm.io/docs/unit-of-work#flush-modes

https://mikro-orm.io/docs/entity-manager#disabling-identity-map-and-change-set-tracking

**Reply by kenspirit** [reply]
@B4nan What is the implicitTransactions option used for?
